etabotsite/etabotsite/settings.py

```diff
@@ -179,6 +179,12 @@
 
 USE_TZ = True
 
+SYS_DOMAIN = ''
+SYS_EMAIL = ''
+SYS_EMAIL_PWD = ''
```

Comment:
let's move password to bash_profile for now and get it via os.environ['ETABOT_ADMIN_EMAIL_PWD'] (or something like that)

Reply:
also let's move SYS_EMAIL, EMAIL_HOST, EMAIL_USE_TLS, EMAIL_PORT to bash_profile or some other settings file away from the code - it will be helpful to test code locally using dummy email accounts and use real one in production